### PR TITLE
Feature/zip handler

### PR DESCRIPTION
**Pull request by KELVINKORAT**
- [x] Created html pages with the same text and rendered it

- [x] Created separate HTML pages

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (vishvaMtech) [inline index.html]: You need to add html files to a folder 'templates', currently it is not able to fetch html pages when executed.

**Review by vishvaMtech — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (KELVINKORAT) [reply index.html]: Ma'am, I have already created the Templates folder inside my PC. I have just uploaded them for review. 
Should I send the folder too ?

Comment (vishvaMtech) [reply index.html]: In order for the code to be successfully executed on any machine, you need to make sure that the html files in a folder named 'templates' in the repository too.

Comment (KELVINKORAT) [reply index.html]: Okay got it.

**Review by vishvaMtech — APPROVED** [review]

**Pull request merged**